packages/notifi-frontend-client/lib/configuration/NotifiFrontendConfiguration.ts

```diff
@@ -203,3 +203,16 @@ export const envUrl = (env: NotifiEnvironment): string => {
       return 'https://api.stg.notifi.network/gql';
   }
 };
+
+export const envWSGraphqlUrl = (env: NotifiEnvironment): string => {
+  switch (env) {
+    case 'Development':
+      return 'ws://api.dev.notifi.network/gql';
+    case 'Local':
+      return 'ws://localhost:5001/gql';
```

Comment:
all of these should be wss instead of ws, except Local

Reply:
ws is equivalent to http
wss is essentially http**s**

Reply:
I think it will be more efficient to combine this section with `envUrl`. Something like below:

```typescript
export const envUrl = (env: NotifiEnvironment, endpointType?: 'websocket' | 'http'): string => {
  switch (env) {
    case 'Development':
      return endpointType === 'websocket' ? 'wss://api.dev.notifi.network/gql' :  'https://api.dev.notifi.network/gql' : '';
    //  others ....
  }
};
```